inc/meta-box-html.php

```diff
@@ -578,17 +596,18 @@ class="tagdelbutton -<?php echo esc_attr( $term->name ) ?>"><?php
 						?></span><?php
 					} else {
 						$date = strtotime( $term->name );
-						?><span><a id="<?php echo esc_attr( $taxonomy ) ?>-check-num-<?php echo esc_attr( $i ) ?>"
-								  class="tagdelbutton -<?php echo esc_attr( $term->name ) ?>"><?php
+						?><span><a id="<?php echo esc_attr( $field['taxonomy'] ) ?>-check-num-<?php echo esc_attr( $i ) ?>"
+								  class="tagdelbutton &<?php echo esc_attr( $term->name ) ?>"><?php
```

Comment:
Just changed L592. Yes that's why I changed it. Before it was splitting by hypens but when you save the data in the ISO8601 format, hypens are present so it breaks.